code/clue.js

```diff
@@ -3,43 +3,273 @@
 const mrGreen = {
   firstName: 'Jacob',
   lastName: 'Green',
-  color: 'green',
-  description: 'He has a lot of connections',
+  color: '#9ba396',
+  description: 'He has a lot of connections.',
   age: 45,
   image: 'assets/green.png',
-  occupation: 'Entrepreneur'
+  occupation: 'Entrepreneur',
+  favouriteWeapon: 'knife'
 }
 
+const professorPlum = {
+  firstName: 'Victor',
+  lastName: 'Plum',
+  color: '#9c96a3',
+  description: 'Has a talent for solving crimes.',
+  age: 35,
+  image: 'assets/plum.png',
+  occupation: 'Professor',
+  favouriteWeapon: 'dumbbell'
+}
+
+const missScarlet = {
+  firstName: 'Cassandra',
+  lastName: 'Scarlet',
+  color: '#a87780',
+  description: 'Young, cunning and attractive.',
+  age: 31,
+  image: 'assets/scarlet.png',
+  occupation: 'Brothel Mami',
+  favouriteWeapon: 'poison'
+}
+
+const mrsPeacock = {
+  firstName: 'Eleanor',
+  lastName: 'Peacock',
+  color: '#7e809c',
+  description: 'Royal-like appearance.',
+  age: 55,
+  image: 'assets/peacock.png',
+  occupation: 'Rich Lady',
+  favouriteWeapon: 'pistol'
+}
+
+const colonelMustard = {
+  firstName: 'Jack',
+  lastName: 'Mustard',
+  color: '#abab89',
+  description: 'Very dangerous and cunning relying on his military training.',
+  age: 65,
+  image: 'assets/mustard.png',
+  occupation: 'Colonel',
+  favouriteWeapon: 'axe'
+}
+
+const mrsWhite = {
+  firstName: 'Mrs',
+  lastName: 'White',
+  color: '#dbdbc1',
+  description: 'Domestic worker and cook.',
+  age: 40,
+  image: 'assets/white.png',
+  occupation: 'Cook',
+  favouriteWeapon: 'candlestick'
+}
+
+
 // CREATE OBJECTS FOR ALL THE WEAPONS, ADD MORE CHARACTERISTICS TO THE WEAPONS IF YOU LIKE.
 
 const rope = {
-  name: 'rope',
-  weight: 10
+  name: 'Rope',
+  weight: 10,
+  color: '#a19c85',
+  mortality: 2,
+  id: 'rope'
```

Comment:
Small thing, but by convention, `id` property should be always the first one in object